adapter/thing/battery.go

```diff
@@ -0,0 +1,43 @@
+package thing
+
+import (
+	"time"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/service/battery"
+	"github.com/futurehomeno/cliffhanger/router"
+	"github.com/futurehomeno/cliffhanger/task"
+)
+
+// BatteryConfig represents a thing configuration.
+type BatteryConfig struct {
+	ThingConfig   *adapter.ThingConfig
+	BatteryConfig *battery.Config
+}
+
+// NewBattery creates a thing that satisfies expectations for the battery.
+func NewBattery(
```

Comment:
Do we have a thing that contains battery service only?

Reply:
It's a possibility from Vinculum.